modules/src/movie.py

```diff
@@ -47,6 +50,12 @@ def process(input, entities):
         template = ButtonTemplate(text)
         template.add_web_url('IMDb Link', 'https://www.imdb.com/title/' + data['imdb_id'] + '/')
 
+	# Append first Trailer URL if one exists
+	for video in videos:
+		if video['type'] == 'Trailer':
+			template.add_web_url('Trailer Link','https://www.youtube.com/watch?v='+video['key']+'/')
```

Comment:
Does it always return a `YouTube` video?

Reply:
I am actually not sure if it will always return a YouTube video but the movies I have tried did return a YouTube video I can look further in to see if there is a way to tell for sure.